packages/@scope/jsii-calc-base/lib/index.ts

```diff
@@ -1,21 +1,34 @@
-import { IVeryBaseInterface, VeryBaseProps } from '@scope/jsii-calc-base-of-base';
+import {
+  IVeryBaseInterface,
+  VeryBaseProps,
+  StaticConsumer as StaticConsumerBase,
+} from '@scope/jsii-calc-base-of-base';
 
 /**
  * A base class.
  */
 export abstract class Base {
-    /**
-     * @returns the name of the class (to verify native type names are created for derived classes).
-     */
-    public typeName() {
-        return (this.constructor as any).name;
-    }
+  /**
+   * @returns the name of the class (to verify native type names are created for derived classes).
+   */
+  public typeName() {
+    return (this.constructor as any).name;
+  }
 }
 
 export interface BaseProps extends VeryBaseProps {
-    readonly bar: string;
+  readonly bar: string;
 }
 
 export interface IBaseInterface extends IVeryBaseInterface {
-    bar(): void;
-}
\ No newline at end of file
+  bar(): void;
+}
+
+/**
+ * Hides the transitive dependency of base-of-base
+ */
+export class StaticConsumer {
```

Comment:
NOTE: this is required to keep `jsii-calc-base-of-base` a transitive dep of `jsii-calc`